charts/reform-scan-blob-router/values.yaml

```diff
@@ -30,6 +30,7 @@ java:
 
     DELETE_DISPATCHED_FILES_CRON: "0 0/10 * * * *"
     HANDLE_REJECTED_FILES_CRON: "0/30 * * * * *"
+    REJECT_DUPLICATES_CRON: "0/30 * * * * *"
```

Comment:
Should we split code and config changes?

Reply:
So just need a chart bump and separate pr from this one ;)

Reply:
@apereverzin 
Why?

Reply:
@rkondratowicz To separate config changes and code changes in order to easier find an issue in case if the build is broken

Reply:
https://github.com/hmcts/blob-router-service/pull/291